helper/testhelpers/docker/testhelpers.go

```diff
@@ -23,3 +27,75 @@ func CleanupResource(t testing.T, pool *dockertest.Pool, resource *dockertest.Re
 	}
 	t.Fatalf("Failed to cleanup local container: %s", err)
 }
+
+func PrepareTestContainer(t *testing.T) (cleanup func(), retAddress, token, mountPath, keyName string, tlsConfig *api.TLSConfig) {
```

Comment:
This func name is pretty generic.  Usually it's used in a package whose name reflects what kind of container we're starting (e.g. consul.PrepareTestContainer) but here the package is `docker`, which doesn't help.  Maybe `PrepareVaultContainer`?

Reply:
changed to `PrepareVaultContainer`